components/Products/Favourite.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+export default class Favourite extends React.Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      favourite: this.props.isFavourite
+    }
+  }
+
+  selectFavourite(){
+    this.setState({
+      favourite: !this.state.favourite
+    });
+  }
+
+  setFavourite() {
```

Comment:
nazwa funkcji nie oddaje co się faktycznie dzieje
To powinno być w renderze i nie używać pngów a jednego svg